tensorflow_addons/metrics/f_scores.py

```diff
@@ -23,72 +23,112 @@
+         ValueError: If the `beta` value is less than or equal
+         to 0.
 
-    1. If `None` is specified as an input, scores for each
-       class are returned.
+      `average` parameter behavior:
 
-    2. If `micro` is specified, metrics like true positivies,
-       false positives and false negatives are computed
-       globally.
+      1. If `None` is specified as an input, scores for each
+         class are returned.
 
-    3. If `macro` is specified, metrics like true positivies,
-       false positives and false negatives are computed for
-       each class and their unweighted mean is returned.
-       Imbalance in dataset is not taken into account for
-       calculating the score
+      2. If `micro` is specified, metrics like true positivies,
+         false positives and false negatives are computed
+         globally.
 
-    4. If `weighted` is specified, metrics are computed for
-       each class and returns the mean weighted by the
-       number of true instances in each class taking data
-       imbalance into account.
+      3. If `macro` is specified, metrics like true positivies,
+         false positives and false negatives are computed for
+         each class and their unweighted mean is returned.
+         Imbalance in dataset is not taken into account for
+         calculating the score
+
+      4. If `weighted` is specified, metrics are computed for
+         each class and returns the mean weighted by the
+         number of true instances in each class taking data
+         imbalance into account.
 
     Usage:
     ```python
-    actuals = tf.constant([[1, 1, 0],[1, 0, 0]],
-              dtype=tf.int32)
-    preds = tf.constant([[1, 0, 0],[1, 0, 1]],
-            dtype=tf.int32)
-    output = tf.keras.metrics.F1Score(num_classes=3,
-              average='micro')
-    output.update_state(actuals, predictions)
-    print('F1 Micro score is: ',
-            output.result().numpy()) # 0.6666667
+    actuals = tf.constant([[1, 1, 0],[1, 0, 0]], dtype=tf.int32)
+    predis = tf.constant([[1, 0, 0],[1, 0, 1]], dtype=tf.int32)
+
+    fb_score = tfa.metrics.FBetaScore(num_classes=3,
+                beta=0.4, average='micro')
+    fb_score.update_state(actuals, preds)
+    print('F1-Beta Score is: ',
+           fb_score.result().numpy()) # 0.6666666
+
+    fb_score = tfa.metrics.FBetaScore(num_classes=3,
+           beta=0.4, average='macro')
+    fb_score.update_state(actuals, preds)
+    print('F1-Beta Score is: ',
+          fb_score.result().numpy()) # 0.33333334
+
+    fb_score = tfa.metrics.FBetaScore(num_classes=3,
+               beta=0.4, average='weighted')
+    fb_score.update_state(actuals, preds)
+    print('F1-Beta Score is: ',
+          fb_score.result().numpy()) # 0.6666667
+
+    fb_score = tfa.metrics.FBetaScore(num_classes=3,
+               beta=0.4, average=None)
+    fb_score.update_state(actuals, preds)
+    print('F1-Beta Score is: ',
+         fb_score.result().numpy()) # [1. 0. 0.]
```

Comment:
Please lint the docstring above. See the docstring of `F1Score` for more details.

Reply:
@SSaishruthi Just need to address this.

Reply:
@Squadrick Thanks for pointing this issue. I misunderstood @WindQAQ comment to be only in this line. Updated now.